app/styles/recap-situation.css

```diff
@@ -0,0 +1,66 @@
+.recap-situation {
+    border-top: none;
+    background-color: white;
+}
+
+.recap-situation h3 {
+    margin-top: 0;
+}
+
+.recap-situation .section-header {
+    margin-top: 21px;
+}
+
+.accordion-ressources .panel {
+    border: none;
+    font-size: 12px;
+}
+
+.accordion-ressources .panel-heading {
+    padding: 5px 8px;
+}
+
+.accordion-ressources .panel-title {
+    font-size: 14px;
+}
+
+.accordion-ressources .panel-body {
+    padding: 10px;
+}
+
+.accordion-ressources hr {
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+.accordion-ressources .recap-indemnites {
+    border-left: 7px solid #AC6C82;
+}
+
+.accordion-ressources .recap-pensions {
+    border-left: 7px solid #685C79;
+}
+
+.accordion-ressources .recap-allocations {
+    border-left: 7px solid #DA727E;
+}
+
+.accordion-ressources .recap-revenusActivite {
+    border-left: 7px solid #FFBC67;
+}
+
+.accordion-ressources .recap-autre {
+    border-left: 7px solid #39747B;
+}
+
+.accordion-ressources .recap-rpns {
+    border-left: 7px solid #8366d4;
+}
+
+.accordion-ressources .recap-patrimoine {
+    border-left: 7px solid #455C7B;
+}
+
+.right-align {
```

Comment:
Never create such non-semantic helper classes, especially when they are so terribly widely scoped.
It is not that we want some random `div` to be right-aligned. It is that, when we display a number in a list, we want it to be right-aligned. This shouldn't leak in the HTML.
The proper way to do this would be to change the markup altogether, but this is out of the scope of this PR.
At least, let's rename that class: which elements does it represent? I guess it is a `ressource-value`, WDYT?